Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -265,6 +265,122 @@ using TrackExtra = TracksExtra::iterator;
+
+// TRACKPARCOVFWD TABLE definition
+DECLARE_SOA_COLUMN(SigmaX, sigmaX, float);
+DECLARE_SOA_COLUMN(SigmaY, sigmaY, float);
+DECLARE_SOA_COLUMN(SigmaPhi, sigmaPhi, float);
+DECLARE_SOA_COLUMN(SigmaTgl, sigmaTgl, float);
+DECLARE_SOA_COLUMN(Sigma1Pt, sigma1Pt, float);
+DECLARE_SOA_COLUMN(RhoXY, rhoXY, int8_t);
+DECLARE_SOA_COLUMN(RhoPhiX, rhoPhiX, int8_t);
+DECLARE_SOA_COLUMN(RhoPhiY, rhoPhiY, int8_t);
+DECLARE_SOA_COLUMN(RhoTglX, rhoTglX, int8_t);
+DECLARE_SOA_COLUMN(RhoTglY, rhoTglY, int8_t);
+DECLARE_SOA_COLUMN(RhoTglPhi, rhoTglPhi, int8_t);
+DECLARE_SOA_COLUMN(Rho1PtX, rho1PtX, int8_t);
+DECLARE_SOA_COLUMN(Rho1PtY, rho1PtY, int8_t);
+DECLARE_SOA_COLUMN(Rho1PtPhi, rho1PtPhi, int8_t);
+DECLARE_SOA_COLUMN(Rho1PtTgl, rho1PtTgl, int8_t);
+
+DECLARE_SOA_EXPRESSION_COLUMN(CXX, cXX, float, aod::trackfwd::sigmaX* aod::trackfwd::sigmaX);
+DECLARE_SOA_EXPRESSION_COLUMN(CXY, cXY, float, (aod::trackfwd::rhoXY / 128.f) * (aod::trackfwd::sigmaX * aod::trackfwd::sigmaY)); //Why /128.f ?
+DECLARE_SOA_EXPRESSION_COLUMN(CYY, cYY, float, aod::trackfwd::sigmaY* aod::trackfwd::sigmaY);
+DECLARE_SOA_EXPRESSION_COLUMN(CPhiX, cPhiX, float, (aod::trackfwd::rhoPhiX / 128.f) * (aod::trackfwd::sigmaPhi * aod::trackfwd::sigmaX));
+DECLARE_SOA_EXPRESSION_COLUMN(CPhiY, cPhiY, float, (aod::trackfwd::rhoPhiY / 128.f) * (aod::trackfwd::sigmaPhi * aod::trackfwd::sigmaY));
+DECLARE_SOA_EXPRESSION_COLUMN(CPhiPhi, cPhiPhi, float, aod::trackfwd::sigmaPhi* aod::trackfwd::sigmaPhi);
+DECLARE_SOA_EXPRESSION_COLUMN(CTglX, cTglX, float, (aod::trackfwd::rhoTglX / 128.f) * (aod::trackfwd::sigmaTgl * aod::trackfwd::sigmaX));
+DECLARE_SOA_EXPRESSION_COLUMN(CTglY, cTglY, float, (aod::trackfwd::rhoTglY / 128.f) * (aod::trackfwd::sigmaTgl * aod::trackfwd::sigmaY));
+DECLARE_SOA_EXPRESSION_COLUMN(CTglPhi, cTglPhi, float, (aod::trackfwd::rhoTglPhi / 128.f) * (aod::trackfwd::sigmaTgl * aod::trackfwd::sigmaPhi));
+DECLARE_SOA_EXPRESSION_COLUMN(CTglTgl, cTglTgl, float, aod::trackfwd::sigmaTgl* aod::trackfwd::sigmaTgl);
+DECLARE_SOA_EXPRESSION_COLUMN(C1PtY, c1PtY, float, (aod::trackfwd::rho1PtY / 128.f) * (aod::trackfwd::sigma1Pt * aod::trackfwd::sigmaY));
+DECLARE_SOA_EXPRESSION_COLUMN(C1PtX, c1PtX, float, (aod::trackfwd::rho1PtX / 128.f) * (aod::trackfwd::sigma1Pt * aod::trackfwd::sigmaX));
+DECLARE_SOA_EXPRESSION_COLUMN(C1PtPhi, c1PtPhi, float, (aod::trackfwd::rho1PtPhi / 128.f) * (aod::trackfwd::sigma1Pt * aod::trackfwd::sigmaPhi));
+DECLARE_SOA_EXPRESSION_COLUMN(C1PtTgl, c1PtTgl, float, (aod::trackfwd::rho1PtTgl / 128.f) * (aod::trackfwd::sigma1Pt * aod::trackfwd::sigmaTgl));
+DECLARE_SOA_EXPRESSION_COLUMN(C1Pt21Pt2, c1Pt21Pt2, float, aod::trackfwd::sigma1Pt* aod::trackfwd::sigma1Pt);
+
+// TRACKFWDEXTRA TABLE definition
+DECLARE_SOA_COLUMN(Chi2, chi2, float);
+DECLARE_SOA_COLUMN(Chi2Match, chi2Match, float);
+//DECLARE_SOA_COLUMN(PDca, pDca, float);
+//DECLARE_SOA_COLUMN(RAtAbsorberEnd, rAtAbsorberEnd, float);
```

Comment:
Didn't you want to store it (and not calculate on the fly)?

Reply:
Indeed, this is also my understanding from what @pillot explained.